### PR TITLE
Added vscode to .gitignore and cleaned up sim mode

### PR DESCRIPTION
**Pull request by christopherdliu**
Just me playing around and testing stuff to make sure things work. First PR yay

### CODE REVIEW

File: Assets/Oculus/Voice/Features/Dictation/Scripts/Runtime/Utilities.meta
Comment (christopherdliu) [inline Assets/Oculus/Voice/Features/Dictation/Scripts/Runtime/Utilities.meta:1]: Not sure what these do, but every time I open up Unity these 3 files get automatically deleted. Everything still runs fine in sim mode at least? Would be nice if I didn't have to undo deleting these 3 every time I open Unity, so figured I'd commit them and we can undo if we run into any problems.

File: Assets/Scripts/GameLogicController.cs [hunk 1]
```diff
@@ -16,49 +16,67 @@ public class GameLogicController : MonoBehaviour
     public GameObject pinPrefab;
     public GameObject ballPrefab;
 
+    // For regular mode
+    private Vector3 CAMERA_START_POSITION = new(4.5f, 1, 0);
```
Comment (anthonyuitz) [inline Assets/Scripts/GameLogicController.cs:20]: this is for sim mode, in regular mode the avatar is initialized based on the start transform that is passed to the thing handling game logic for the example scene

File: Assets/Scripts/GameLogicController.cs [hunk 2]
```diff
@@ -16,49 +16,67 @@ public class GameLogicController : MonoBehaviour
...
+    private void _ResetPinsAndBall()
+    {
+        Destroy(_pinParent);
+        Destroy(_currentBall);
+        _pinParent = Instantiate(pinPrefab, PINS_START_POSITION, Quaternion.identity);
+        _currentBall = Instantiate(ballPrefab, BALL_START_POSITION, Quaternion.identity);
+    }
+
+
+    private void _ResetPinsAndBallSimMode()
```
Comment (anthonyuitz) [inline Assets/Scripts/GameLogicController.cs:75]: given the overlap of these two methods, could just potentially make an if statement inside the original method as sim mode is a global variable

File: Assets/Oculus/Voice/Features/Dictation/Scripts/Runtime/Utilities.meta
Comment (anthonyuitz) [reply Assets/Oculus/Voice/Features/Dictation/Scripts/Runtime/Utilities.meta:1]: These are how Unity tracks files for things like references on GameObjects. They should only be auto-deleted when the original file doesn't exist. Which it doesn't. So there must have been a package update or something?

**Pull request merged**